lib/emscripten/src/syscalls/unix.rs

```diff
@@ -130,7 +130,18 @@ extern "C" {
     pub fn lstat64(path: *const libc::c_char, buf: *mut c_void) -> c_int;
 }
 
-#[cfg(not(target_os = "macos"))]
+// Linking to functions that are not provided by rust libc
```

Comment:
Is this permanent? If not, I'd really appreciate it if this could include a link to an issue or PR.

Reply:
I can try to upstream the fn definition.
There is a PR for the ucontext/mcontext part https://github.com/rust-lang/libc/pull/1630